Description: fix FTBFS of #634401 regarding opaque_ext2_group_desc
  extundelete: FTBFS: extundelete.cc:963:47: error:
   invalid use of incomplete type 'struct opaque_ext2_group_desc'
Forwarded: yes
Author: Eric Sandeen <sandeen@redhat.com>
Last-Update: 2011-01-11

--- a/src/extundelete.cc
+++ b/src/extundelete.cc
@@ -941,7 +941,7 @@ int load_super_block(ext2_filsys fs)
 	group_descriptor_table = new ext2_group_desc[groups_];
 	for (uint32_t n = 0; n < fs->group_desc_count; n++)
 	{
-		group_descriptor_table[n] = fs->group_desc[n];
+		group_descriptor_table[n] = *ext2fs_group_desc(fs, fs->group_desc, n);
 	}
 	return errcode;
 }
